test/integration/YaoBlocks_test.jl

```diff
@@ -8,35 +8,35 @@
     @test issetequal(sites(tn), [site"1", site"2", site"1'", site"2'"])
     @test Tenet.ntensors(tn) == 2
 
-    @testset "GHZ Circuit" begin
-        circuit_GHZ = chain(n_qubits, put(1 => Yao.H), Yao.control(1, 2 => Yao.X), Yao.control(2, 3 => Yao.X))
+    # @testset "GHZ Circuit" begin
```

Comment:
```suggestion
    @test_skip begin #@testset "GHZ Circuit"
```

Reply:
I added a new `@testset_skip` macro for this.